cookiecutter/{{cookiecutter.github_repo_name}}/pyproject.toml

```diff
@@ -38,6 +38,7 @@ dependencies = [ # TODO(cookiecutter): Remove and add more if appropriate
 ]
 {%- elif cookiecutter.type == "api" %}
 dependencies = [ # TODO(cookiecutter): Remove and add more if appropriate
+  "frequenz-api-common >= 0.2.0, < 0.3.0",
```

Comment:
Same question.
Not sure why we need the submodule here.

Reply:
also this is no longer possible right?  wasn't the python code removed from `frequenz-api-common`?  I guess if `cookiecutter.type == "api"`, just add `frequenz-api-common` as a submodule should be enough

Reply:
OK, let me make some basic clarification, just in case, to make sure we all have a common understanding (sorry if it was clear before): This code is a template to generate new API repos. Let's say for now we are generating a repo frequenz-api-lala.

So going to you questions:

* @shsms, frequenz-api-common still provides a Python package, for the same reasons I will explain here :)

* We need the Python dependency here because I assume frequenz-api-lala will use frequenz-api-common, and if it does, the generate python code will also have some import frequenz.api.common.

* Remember that we can't generate the frequenz.api.package in each downstream project, like frequenz-api-lala, because otherwise we'll have multiple definitions of the same package.

Reply:
Ok, I see. It makes sense now. Thanks for the explanation.

>  frequenz-api-lala

I love it. Let's go with this name for the next API.